manifests/g/GNE/DualMonitorTools/2.7.0.0/GNE.DualMonitorTools.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: GNE.DualMonitorTools
+PackageVersion: 2.7.0.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral

```

Comment:
```suggestion
  Architecture: x86
```

Reply:
Not sure how it got set to neutral, unless something else is selected by default I choose x86